external_server/adapters/mqtt/adapter.py

```
@@ -0,0 +1,404 @@
+
+import paho.mqtt.client as mqtt
+from paho.mqtt.client import (
+    Client as _Client,
+    _ConnectionState as _ConnectionState,
+    error_string as _error_string,
+    MQTTErrorCode as _MQTTErrorCode,
+    MQTTMessage as MQTTMessage,
+)
+from external_server.logs import CarLogger as _CarLogger
+from external_server.checkers.mqtt_session import MQTTSession
+from external_server.checkers.mqtt_session import MQTTSession
```

Ruff

Redefinition of unused `MQTTSession` from line 21

Remove definition: `MQTTSession`

(F811)